### PR TITLE
Update the Portfolio to the desktop version

### PR DESCRIPTION
**Pull request by camilevahviraki**
### **In this desktop version of the portfolio**:

**We added :**
  - **style_desktop.css** file witch contain the design of the portfolio to fit the desktop screen.
  - Edited the style of index.html desktop format.
  
**-Updated the index.html file:**

  -  change images url links to use images for the desktop.
  -  put the`<section class="section_M>` inside a grid container.
 
**- Added images inside of images_desk folder**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -20,6 +20,10 @@ nav {
   box-sizing: border-box;
 }
 
+.nav_list {
+  display: none;
+}
+
```
Comment (OLIPLICHE) [inline index.css:26]: - [ ] Please use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider). Open this [Link](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) to Learn more about media queries

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @camilevahviraki , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Linter checks pass
✔️ Descriptive README
✔️ Descriptive PR

 ## Required Changes ♻️

- [ ] Please, as the previous review requested, make your desktop version look like the template you selected (Desktop: 768px or wider). I Provided the Link about "media queries" in the inline comments.
- [ ] Please as it was requested from the previous review that your desktop version needs to look like the template you selected. Kindly implement this below: 
- From Your Design Template:
a

- From Figma Template:
b

- **Please Make sure the `See Project` button pointed in the picture takes you to the GitHub repository** ✔️.

- [ ] CHECK THE COMMENTS UNDER THE REVIEW

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] **Please kindly consider this**:
- Make sure, when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears to the bottom of the image and takes the whole image width

c



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css (hunk 1 above)
Comment (camilevahviraki) [reply index.css:26]: I used Media query , I don't know what you suggest  me again.

Comment (camilevahviraki) [reply index.css:26]: Please Your comment is not really clear, I did what the previous reviewer said Is just I forgot the border beside the Image
Kindly your explanations will be helpful

Comment (camilevahviraki) [reply index.css:26]: Look the design with **screen: 1440px** the design will be exactly the same as in the template.

**Review by Deepakdanger — APPROVED** [review]
Hi @camilevahviraki,

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**